napari/_app_model/constants/_menus.py

```diff
@@ -42,13 +149,10 @@ class LAYERLIST_CONTEXT:
         LINK = '9_link'
 
 
-# TODO: add these to docs, with a lookup for what each menu is/does.
-_CONTRIBUTABLES = {MenuId.LAYERLIST_CONTEXT.value}
-"""Set of all menu ids that can be contributed to by plugins."""
-
-
 def is_menu_contributable(menu_id: str) -> bool:
     """Return True if the given menu_id is a menu that plugins can contribute to."""
     return (
-        menu_id in _CONTRIBUTABLES if menu_id.startswith("napari/") else True
+        menu_id in MenuId.contributables()
+        if menu_id.startswith("napari/")
+        else True
```

Comment:
```suggestion
        menu_id in MenuId.contributables() or not menu_id.startswith("napari/")
```

Reply:
id rather this be expanded to a full-blown if-else statement with comments rather than a confusing one-liner

Reply:
ok